server_status/tests/settings.py

```diff
@@ -11,32 +11,89 @@
 """
 from __future__ import unicode_literals
 from distutils.version import StrictVersion  # pylint: disable=import-error, no-name-in-module
+import os
+import logging
 
 import django
 
-from server_status.tests.test_settings import *  # pylint: disable=wildcard-import, unused-wildcard-import
+DEBUG = True
 
+logging.getLogger("factory").setLevel(logging.WARN)
 
-django_version = django.get_version()  # pylint: disable=invalid-name
-if StrictVersion(django_version) < StrictVersion('1.7'):
-    INSTALLED_APPS.append('south', )
+SITE_ID = 1
 
+APP_ROOT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..'))
 
-BROKER_URL = 'redis://redis:6379/4'
-CELERY_RESULT_BACKEND = 'redis://redis:6379/4'
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'postgres',
         'USER': 'postgres',
         'PASSWORD': '',
-        'HOST': 'db',
+        'HOST': os.environ.get('DATABASE_HOST', '127.0.0.1'),
         'PORT': '5432',
     }
 }
-HAYSTACK_CONNECTIONS = {
-    'default': {
-        'URL': 'http://elastic:9200',
-        'INDEX_NAME': 'haystack',
-    }
-}
+
+ROOT_URLCONF = 'server_status.tests.urls'
+
+STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(APP_ROOT, '../app_static')
+MEDIA_ROOT = os.path.join(APP_ROOT, '../app_media')
+STATICFILES_DIRS = (
+    os.path.join(APP_ROOT, 'static'),
+)
+
+TEMPLATE_DIRS = (
+    os.path.join(APP_ROOT, 'tests/test_app/templates'),
+)
+
+EXTERNAL_APPS = [
+    'django.contrib.admin',
+    'django.contrib.admindocs',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.messages',
+    'django.contrib.sessions',
+    'django.contrib.staticfiles',
+    'django.contrib.sitemaps',
+    'django.contrib.sites',
+]
+
+INTERNAL_APPS = [
+    'server_status',
+    'server_status.tests.test_app',
+]
+
+INSTALLED_APPS = EXTERNAL_APPS + INTERNAL_APPS
+
+MIDDLEWARE_CLASSES = [
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+]
+
+COVERAGE_REPORT_HTML_OUTPUT_DIR = os.path.join(
+    os.path.join(APP_ROOT, 'tests/coverage'))
+COVERAGE_MODULE_EXCLUDES = [
+    'tests$', 'settings$', 'urls$', 'locale$',
+    'migrations', 'fixtures', 'admin$', 'django_extensions',
+]
+COVERAGE_MODULE_EXCLUDES += EXTERNAL_APPS
+
+SECRET_KEY = 'foobar'
+
+ELASTICSEARCH_URL = os.environ.get('ELASTICSEARCH_URL', 'http://127.0.0.1:9200')
+
+USE_CELERY = True
+STATUS_TOKEN = 'asdf'
+
+BROKER_URL = os.environ.get('BROKER_URL', 'redis://127.0.0.1:6379/4')
```

Comment:
Is there a particular reason that the broker URL is defaulting to database 4 in this case? My understanding is that Redis DBs usually start at 0.